cpp/src/arrow/flight/types.cc

```diff
@@ -45,6 +45,12 @@ std::string FlightStatusDetail::ToString() const { return CodeAsString(); }
 
 FlightStatusCode FlightStatusDetail::code() const { return code_; }
 
+std::string FlightStatusDetail::ExtraInfo() const { return extra_info_; }
```

Comment:
These are simple getter/setters so they should be named `extra_info` and `set_extra_info`.

Reply:
:+1: